client/src/templates/Challenges/classic/show.tsx

```diff
@@ -400,11 +399,7 @@ class ShowClassic extends Component<ShowClassicProps, ShowClassicState> {
         usesMultifileEditor={usesMultifileEditor}
       >
         <LearnLayout>
-          <Helmet
-            title={`${t(
-              'learn.learn'
-            )} ${this.getBlockNameTitle()} | freeCodeCamp.org`}
-          />
+          <Helmet title={`${this.getBlockNameTitle()} | freeCodeCamp.org`} />
```

Comment:
```suggestion
          <Helmet title={`${this.getBlockNameTitle()} | freeCodeCamp.org`} />
```
Out of scope for this PR, but shouldn't these be translated?

Reply:
freeCodeCamp.org isn't translated.

<img width="208" alt="image" src="https://user-images.githubusercontent.com/63889819/143077271-1b0a35c1-b695-4058-9e5f-9910e81d5299.png">

The block names are translated elsewhere (I assume via the frontmatter).

Reply:
It should be "HTML básico y HTML5", no?

Reply:
Feel free to mark this resolved whenever you want to merge, Nick.  I just wanted to mention the issue before I forgot about it.

Reply:
Oh snap yes you're correct.

Reply:
I went ahead and fixed the translation issue in this PR too, since it's touching the same code.